packages/components/menu/Menu.mdx

```diff
@@ -331,19 +332,6 @@ But you can redefine them on any submenu by passing those props directly to that
 </Menu>
 ```
 
-## Content recommendations
-
-- Use an interactive element such as `button` for `Menu.Trigger`
-
-## Accessibility
-
-- When the menu is opened, focus is moved to the first menu item. If you set `autoFocus`to `false`, it will not move the focus.
-- When the menu is closed, focus returned back to the trigger element.
-- You can use arrow keys (`ArrowUp` and `ArrowDown`) to navigate through menu items.
-- When the menu is open, click on `Esc` key will close the popover. If you set `closeOnEsc` to `false`, it will not close.
-- When the menu is open, click outside menu or blurring out will close the menu. If you set `closeOnBlur` to `false`, it will not close.
-- All the necessary a11y attributes for `Menu.List`, `Menu.Trigger` and `Menu.Item` are provided.
-
 ## Props
```

Comment:
Did you use the plop templates to make this convertion?

This title here is supposed to be some react code because we need both a title and a textlink aligned

The code would be something like this:
```suggestion
<Stack justifyContent="space-between" marginTop="spacing2Xl" marginBottom="spacingL">
  <Heading id="props-api-reference" as="h2" marginTop="none" marginBottom="none">
    Props (API reference)
  </Heading>

  <TextLink
    href="https://v4-f36-storybook.netlify.app/?path=/story/components-menu"
    target="_blank"
    alignIcon="end"
    icon={<ExternalLinkIcon />}
  >
    Open in Storybook
  </TextLink>
</Stack>
```

But of course you also need to import these components on the top of this MDX files or the page will break
[Here is an example of mdx using this approach](https://github.com/contentful/forma-36/blob/forma-v4/packages/components/accordion/Accordion.mdx)

Reply:
I didn't use the plop template 😬 I will. thanks!